src/ReactiveUI.Validation/Helpers/ReactiveValidationObject.cs

```diff
@@ -60,25 +60,39 @@ protected ReactiveValidationObject(IScheduler scheduler = null)
         /// <inheritdoc />
         public virtual IEnumerable GetErrors(string propertyName)
         {
-            var memberInfoName = GetType()
-                .GetMember(propertyName)
-                .FirstOrDefault()?
-                .ToString();
+            //https://stackoverflow.com/questions/34993261/when-is-inotifydataerrorinfo-geterrors-called-with-null-vs-string-empty
```

Comment:
Leaving this in the PR "Other Information" section is sufficient for documentation.  Please remove this comment.

Reply:
Sorry - removed that line